src/app/games/conspirateurs/tests/conspirateurs.component.spec.ts

```diff
@@ -0,0 +1,161 @@
+        expect(testUtils.getComponent()).withContext('Component should be created').toBeTruthy();
+    });
+    describe('drop phase', () => {
+        it('should allow drops at the beginning of the game with a simple click', fakeAsync(async() => {
+            // Given the initial state
+            // When clicking in the central zone
+            // Then a drop is made
+            const move: ConspirateursMove = ConspirateursMoveDrop.of(new Coord(7, 7)).get();
+            await testUtils.expectMoveSuccess('#click_7_7', move);
+        }));
+        it('should forbid dropping outside of the central zone', fakeAsync(async() => {
+            // Given the initial state
+            // When clicking out of the central zone
+            // Then an error is shown
+            const move: ConspirateursMove = ConspirateursMoveDrop.of(new Coord(0, 0)).get();
+            await testUtils.expectMoveFailure('#click_0_0', ConspirateursFailure.MUST_DROP_IN_CENTRAL_ZONE(), move);
+        }));
+    });
+    describe('move phase', () => {
+        beforeEach(() => {
+            // Given a state after the drop phase
+            const state: ConspirateursState = new ConspirateursState([
+                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, B, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, B, _, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, A, _, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, B, _, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+                [_, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _],
+            ], 42);
+            testUtils.setupState(state);
+        });
+        it('should not allow selecting an empty space', fakeAsync(async() => {
+            // When clicking on an empty space
+            // Then the click is rejected
+            await testUtils.expectClickFailure('#click_0_0', RulesFailure.MUST_CHOOSE_PLAYER_PIECE());
+        }));
+        it('should allow performing a simple move by clicking on a piece and then on its destination', fakeAsync(async() => {
+            // Given that a player piece is selected
```

Comment:
Déjà dit dans une autre PR mais "given that" c'est pas plutôt "étant donné que" et pas "ayant ceci" ?

Reply:
changé